src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -137,6 +138,12 @@ private AMQP.BasicProperties getEnrichedProperties(AMQP.BasicProperties properti
         if (properties.getHeaders() != null) {
             enrichedHeaders.putAll(properties.getHeaders());
         }
+        var retryCount = 1;
```

Comment:
Should we call this publishCount (given that it is starting with 1 even during 1st time publish) ?